numpyro/contrib/forecast/gt.py

```diff
@@ -0,0 +1,285 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+from jax import lax, nn
+import jax.numpy as np
+
+import numpyro
+import numpyro.distributions as dist
+
+
+# src: https://github.com/cbergmeir/Rlgt/blob/master/Rlgt/R/rlgtcontrol.R
+DEFAULT_HYPERPARAMETERS = {
+    "cauchy_sd_div": 150,
+    "min_nu": 2,
+    "max_nu": 20,
+    "min_pow_trend": -0.5,
+    "max_pow_trend": 1,
+    "pow_trend_alpha": 1,
+    "pow_trend_beta": 1,
+    "pow_season_alpha": 1,
+    "pow_season_beta": 1,
+    "min_sigma": 1e-10,
+    "min_val": 1e-30,
+    "max_val": 1e38,
+}
+
+
+class GlobalTrendModel:
+    def __init__(self,
```

Comment:
nit: now that we're python 3 only, I've have started using the `*,` separator to force kwargs. Does this work with NumPyro? If so you could define this as
```py
class GlobalTrendModel:
    def __init__(self, *,
                 seasonality=1,
                 ...
                 hyperparameters={}):
```
This allows us to freely reorder parameters in the future, while maintaining backwards compatibility.

Reply:
Thanks, I'll do it and add a backtest function too.